.github/workflows/release.yml

```
@@ -0,0 +1,32 @@
+---
+name: Release new version
+on:
+    release:
+        types: [created]
+
+jobs:
+    test:
+        runs-on: ubuntu-20.04
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-python@v2
+              with:
+                  python-version: 3.8
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: '12.x'
+            - run: make test
+
+    publish-npm:
+        needs: test
+        runs-on: ubuntu-20.04
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v2
+              with:
+                  node-version: 12
+                  registry-url: https://registry.npmjs.org/
+            - run: yarn
+            - run: npm publish
```

per https://docs.github.com/en/actions/guides/publishing-nodejs-packages#publishing-packages-using-yarn

```suggestion
            - run: yarn publish
```